pgmpy/readwrite/BIF.py

```diff
@@ -0,0 +1,312 @@
+import numpy as np
```

Comment:
@khalibartan Please check PEP8 way of ordering and grouping imports. https://www.python.org/dev/peps/pep-0008/#imports